components/button/button.html

```diff
@@ -0,0 +1,206 @@
+              ></path>
+            </g>
+          </svg>
+        </nav>
+      </header>
+
+      <div class="main-body">
+        <aside class="main-sidebar">
+          <span class="sidebar-header">Components</span>
+          <ul class="sidebar-items">
+            <a href="../alert/alert.html" class="sidebar-item">Alert</a>
+            <a href="../avatar/avatar.html" class="sidebar-item">Avatar</a>
+            <a href="../badge/badge.html" class="sidebar-item">Badge</a>
+            <a href="#" class="sidebar-item active">Button</a>
+            <a href="../card/card.html" class="sidebar-item">Card</a>
+            <a href="../image/image.html" class="sidebar-item">Image</a>
+            <a href="../input/input.html" class="sidebar-item">Input</a>
+          </ul>
+        </aside>
+        <div class="main-content-container">
+          <div class="main-content">
+            <h1 class="component-title">Button</h1>
+            <p class="component-info">
+              The Button component is used to trigger an action or event, such
+              as submitting a form, opening a dialog, canceling an action, or
+              performing a delete operation.
+            </p>
+            <div class="component-type">Button Types</div>
+
+            <div class="component-example">
+              <div class="button-container">
+                <button class="btn primary">Primary</button>
+                <button class="btn secondary">Secondary</button>
```

Comment:
I think secondary doesn't have the hover effect being applied due to same color

Reply:
Yea i was going to say same

Reply:
I have intentionally not given any hover to secondary.